README.md

```diff
@@ -17,6 +17,30 @@ import { dsnp } from "frequency-schemas";
 console.log(dsnp.getSchema("broadcast"));
 ```
 
+### Get Schema Id from Chain
+
+```typescript
+import { dsnp } from "frequency-schemas";
```

Comment:
blocking: need to prefix with @dsnp 
```suggestion
import { dsnp } from "@dsnp/frequency-schemas";
```

Reply:
Committed that change. Was that the issue?

Reply:
@wesbiggs Are you sure you committed the change here? I don't see it.

Reply:
Good catch, fixed.